doc/rados/operations/stretch-mode.rst

```diff
@@ -157,28 +157,28 @@ possible, if needed).
 
 .. _Changing Monitor elections: ../change-mon-elections
 
-Stretch Mode Limitations
-========================
-As implied by the setup, stretch mode only handles 2 sites with OSDs.
+Limitations of Stretch Mode 
+===========================
+When using stretch mode, OSDs must be located at exactly two sites. 
 
-While it is not enforced, you should run 2 monitors in each site plus
-a tiebreaker, for a total of 5. This is because OSDs can only connect
-to monitors in their own site when in stretch mode.
+Two monitors plus a tiebreaker monitor should be run in each data center, for a
```

Comment:
For me this part sounds like we need 2 + 1 (arbiter) in each data center. I think we should say something like "Two monitors in each data center, plus the a fifth being a tiebreak monitor".

Reply:
VV

Reply:
@kamoltat's suggestion here is covered by @anthonyeleven's revision above.